src/tests/ftest/osa/osa_online_reintegration.py

```diff
@@ -282,3 +297,13 @@ def test_osa_online_reintegration(self):
         # Perform reintegration testing with 1 pool.
         for pool_num in range(1, 2):
             self.run_online_reintegration_test(pool_num)
+
+    def test_osa_online_reintegration_server_stop(self):
+        """Test ID: DAOS-5920.
+
+        Test Description: Validate Online Reintegration with server stop
+
+        :avocado: tags=all,pr,hw,large,osa,online_reintegration,DAOS_5610
+        """
+        # Perform reintegration testing with 1 pool.
+        self.run_online_reintegration_test(1, server_boot=True)
```

Comment:
(pylint-missing-final-newline) Final newline missing
(pylint-mixed-line-endings) Mixed line endings LF and CRLF